lib/media/track/localmediatrack.js

```diff
@@ -187,10 +187,10 @@ function restartWhenInadvertentlyStopped(localMediaTrack) {
   }
 
   document.addEventListener('visibilitychange', handleTrackStateChange);
-  localMediaTrack.on('stopped', handleTrackStateChange);
+  localMediaTrack.addListener('stopped', handleTrackStateChange);
```

Comment:
Nitpick: We use `on` to attach listeners.
```suggestion
  localMediaTrack.on('stopped', handleTrackStateChange);
```